corehq/apps/sms/models.py

```diff
@@ -266,8 +266,8 @@ def to_json(self):
         return data
 
     def publish_change(self):
-        from corehq.apps.sms.tasks import publish_sms_change
-        publish_sms_change.delay(self)
+        from corehq.apps.sms.change_publishers import publish_sms_saved
+        publish_sms_saved(self)
```

Comment:
we should wrap this in a try/except so that if kafka is down it doesn't interrupt the process